### PR TITLE
'sesdev replace-mgr-modules' should also replace 'bin/cephadm'

### PR DESCRIPTION
**Pull request by ricardoasmarques**
In addition to replace MGR modules, 'bin/cephadm' should also be replaced in order to test cephadm patches / PRs.

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

File: seslib/deployment.py [hunk 1]
```diff
@@ -1210,9 +1214,37 @@ def replace_mgr_modules(self, local=None, pr=None, branch=None, repo=None, langs
                 .format(node_version, master_path, langs, npm_build))
             tools.run_sync(ssh_cmd)
 
-        for node in mgr_nodes:
+        # Fetch bin/cephadm
+        if self.settings.version not in ['nautilus', 'ses6']:
```
Comment (smithfarm) [inline seslib/deployment.py:1218]: `['ses5', 'nautilus', 'ses6']`

but maybe this would be better:

```
if self.settings.deployment_tool == 'cephadm':
```

Comment (ricardoasmarques) [reply seslib/deployment.py]: Done

**Comment by smithfarm** [comment]
This changeset is limited to the `replace_mgr_modules` method, which I personally do not use or test.

@bk201 @tspmelo Do you care to give this a spin?

**Comment by sebastian-philipp** [comment]
sounds like we duplicate cstart here?

**Comment by smithfarm** [comment]
> sounds like we duplicate cpatch to some degree here?

Doesn't cpatch build a container image, though? This sesdev feature (`replace-mgr-modules`) only copies files into a running container using `podman cp`. It does not build a container image.

**Comment by ricardoasmarques** [comment]
> lgtm! Just want to mentioned the cepahdm binary inside mgr container is not replaced. And that one is used by cephadm module in root mode.

Good catch, I'll fix it.

**Comment by ricardoasmarques** [comment]
> > lgtm! Just want to mentioned the cepahdm binary inside mgr container is not replaced. And that one is used by cephadm module in root mode.
> 
> Good catch, I'll fix it.

Done

**Comment by bk201** [comment]
> > > lgtm! Just want to mentioned the cepahdm binary inside mgr container is not replaced. And that one is used by cephadm module in root mode.
> > 
> > 
> > Good catch, I'll fix it.
> 
> Done

Tested again and it works! Thanks.

**Pull request merged**